tasks/run.yml

```diff
@@ -4,9 +4,12 @@
     src: "templates/nifi.service.j2"
     dest: "/etc/systemd/system/nifi.service"
 
-- name: Re/Start NiFi
+- name: Flush Handlers
```

Comment:
What is the purpose of this task?

Reply:
Hi @cavemandaveman.
This calls the handler (which will restart NiFi if anything is changed) before the "Start Nifi" task below.
This is to stop the case (initial install) when NiFi will be installed and started, and then restarted directly afterwards.

I hope that makes sense.  We could probably rename it.  I just got the naming convention from ansibles website.
https://docs.ansible.com/ansible/latest/user_guide/playbooks_handlers.html#controlling-when-handlers-run